src/main/java/mate/academy/patternsOnLesson/BuilderPattern.java

```diff
@@ -0,0 +1,45 @@
+package mate.academy.patternsOnLesson;
+
+public class BuilderPattern implements Pattern {
+    private String name;
+    private int age;
+
+    public BuilderPattern(String name, int age) {
```

Comment:
Should this one need to be public?

Reply:
Maybe no) Inner class use this constructor, so this one can be "privat"(I changed to privat)